src/System.Runtime/tests/System.Runtime.Tests.builds

```diff
@@ -2,7 +2,12 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Runtime.Tests.csproj"/>
+    <Project Include="System.Runtime.Tests.csproj">
+      <TargetGroup>netstandard1.5</TargetGroup>
+    </Project>
+    <Project Include="System.Runtime.Tests.csproj">
```

Comment:
The default is now netstandard1.7 so that is the one without targetgroup, and the one with TargetGroup is the old one